scripts/keystone_data.sh

```diff
@@ -194,9 +194,9 @@ if [[ "$ENABLED_SERVICES" =~ "n-cpu" ]]; then
     get_or_create_endpoint \
         "placement" \
         "RegionOne" \
-        "http://$SERVICE_HOST:8780/" \
-        "http://$SERVICE_HOST:8780/" \
-        "http://$SERVICE_HOST:8780/"
+        "http://$SERVICE_HOST:8778/" \
```

Comment:
I took the install guide to be correct because we are the only ones not running it on port 8778:

UCA: http://paste.suse.de/25095
RDO: https://review.rdoproject.org/r/gitweb?p=openstack/nova-distgit.git;a=blob;f=nova-placement-api.conf;h=fe8c3ad27a9a8e83c5bcf027219f3f8265f457f9;hb=HEAD#l1

Devstack does run it on 8780 http://git.openstack.org/cgit/openstack-dev/devstack/tree/files/apache-placement-api.template#n5 but the comment in that file makes it seem like they regret doing so.

Nova itself does not have any preference or example that I can find.

Yeah, not quite sure what the best way forward is.